docs/sections/papers/deita.md

```diff
@@ -0,0 +1,530 @@
+
+Score, Evolved Response:
+(4.0, 'Here are three essential tips for maintaining good health: \n1. Prioritize regular exercise \n2. Eat a balanced diet with plenty of fruits and vegetables \n3. Get an adequate amount of sleep each night.')
+(2.0, 'Here are three effective strategies to maintain a healthy lifestyle.')
+(5.0, 'Here are three practical tips to maintain good health: Ensure a balanced diet, engage in regular exercise, and prioritize sufficient sleep. These practices support overall well-being.')
+
+```
+
+## 4. Improving Data Diversity
+
+One main component of good data to instruct-tune LLMs is diversity. Real world data can often contain [redundancy](https://openreview.net/forum?id=u96ZBg_Shna) due repetitive and homogenous data.
+
+The authors of the DEITA paper tackle the challenge of ensuring data
+diversity in the instruction tuning LLMs to avoid the pitfalls of data
+redundancy that can lead to overfitting or poor generalization. They
+propose an embedding-based method to filter data for diversity. This
+method, called Repr Filter, uses embeddings generated by the LLaMA-1 13B
+model to represent instruction-response pairs in a vector space. The
+diversity of a new data sample is assessed based on the cosine distance
+between its embedding and that of its nearest neighbor in the already
+selected dataset. If this distance is greater than a specified
+threshold, the sample is considered diverse and is added to the
+selection. This process prioritizes diversity by assessing each
+sample\'s contribution to the variety of the dataset until the data
+selection budget is met. This approach effectively maintains the
+diversity of the data used for instruction tuning, as demonstrated by
+the DEITA models outperforming or matching state-of-the-art models with
+significantly less training data.
+
+![image.png](https://github.com/argilla-io/distilabel/blob/docs/deita-tutorial/docs/assets/tutorials-assets/deita/diversity.png)
+
+```python
+generate_conversation = ConversationTemplate(
+    name="generate_conversation",
+    input_mappings={
+        "instruction": "evolved_instruction",
+        "response": "evolved_response",
+    },
+    pipeline=pipeline,
+)
+
+generate_embeddings = GenerateEmbeddings(
+    name="generate_embeddings",
+    llm=TransformersLLM(
+        model="TinyLlama/TinyLlama-1.1B-Chat-v1.0",
+        device="cuda",
+        torch_dtype="float16",
+    ),
+    input_mappings={"text": "conversation"},
+    input_batch_size=5,
+    pipeline=pipeline,
+)
+
+deita_filtering = DeitaFiltering(name="deita_filtering", pipeline=pipeline)
+```
+# Build the ⚗ distilabel `Pipeline`
+
+Now we\'re ready to build a distilabel 1.0 pipeline using the DEITA
```

Comment:
Maybe we could remove all the version identifiers within the text and just pin the dependency instead to e.g. `==1.0.0`